server/helper.py

```
@@ -290,13 +293,55 @@ def write_file(pPath, pText):
+        except KeyError:
+            mylog('none', [f'[Settings] ⚠ ERROR - setting_missing - Setting not found for key: {key} in cached file {settingsFile}'])  
+    
+    LASTCACHEDATE = fileModifiedTime
+    
+    try:
+        with open(settingsFile, 'r') as json_file:
+            data = json.load(json_file)
+            for item in data.get("data",[]):
+                # map setkey to item
+                CACHE[item["setKey"]] = item
+    except (FileNotFoundError, json.JSONDecodeError, ValueError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)